### ISSUE TITLE
"Missing ember data" When Using ember-fill Commmand

### ISSUE THREAD

**Issue by charlie-e09ae35d** [RootsTeam/Embers#71]
multimc 0.4.11-703
forge 12.18.3.2185
embers 0.103

Despite Atmospheric Gauge indicating non zero ember level any "ember-fill" command switch (add, set, query) returns "Missing ember data at chunk..."

**Comment by elucent** [comment]
I believe I broke the command when I changed how ember generated. It'll be a bit of a pain to fix, but I'll give it a shot.

**Comment by jco2641** [comment]
I've been looking at the code for a bit before I came to report the same thing.
- I can't see why if there is a value for the current chunk that the gauge would see it but the command wouldn't, unless there is something caused by the parseDouble() call that only the command uses.  But maybe abstracting this hash key generation could be a useful refactor.
- if the command sets a value, I don't think it will correctly save, because the command doesn't tell the EmberWorldData that its hashmap has been updated by calling the markDirty() method.
